Maddori.Apple/Maddori.Apple/Screen/AddFeedbackContent/AddFeedbackContentViewController.swift

```diff
@@ -0,0 +1,69 @@
+//
+//  AddFeedbackContentViewController.swift
+//  Maddori.Apple
+//
+//  Created by 김유나 on 2022/10/19.
+//
+
+import UIKit
+
+import SnapKit
+
+final class AddFeedbackContentViewController: BaseViewController {
+    
+    // MARK: - property
+
+    private let backButton = BackButton()
+    private let closeButton = CloseButton()
+    private let addFeedbackTitleLabel: UILabel = {
+        let label = UILabel()
+        label.text = TextLiteral.addFeedbackContentViewControllerTitleLabel
+        label.textColor = .black100
+        label.font = .title
+        return label
+    }()
+    private let feedbackTypeLabel: UILabel = {
+        let label = UILabel()
+        label.text = TextLiteral.addFeedbackContentViewControllerFeedbackTypeLabel
+        label.textColor = .black100
+        label.font = .label2
+        return label
+    }()
+    private let feedbackTypeButtonView = FeedbackTypeButtonView()
+    
+    // MARK: - lifecycle
+    
+    override func render() {
+        view.addSubview(addFeedbackTitleLabel)
+        addFeedbackTitleLabel.snp.makeConstraints {
+            $0.top.equalTo(view.safeAreaLayoutGuide).offset(SizeLiteral.topPadding)
+            $0.leading.equalToSuperview().offset(SizeLiteral.leadingTrailingPadding)
+        }
+        
+        view.addSubview(feedbackTypeLabel)
+        feedbackTypeLabel.snp.makeConstraints {
+            $0.top.equalTo(addFeedbackTitleLabel.snp.bottom).offset(40)
+            $0.leading.equalToSuperview().inset(SizeLiteral.leadingTrailingPadding)
+        }
+        
+        view.addSubview(feedbackTypeButtonView)
+        feedbackTypeButtonView.snp.makeConstraints {
+            $0.top.equalTo(feedbackTypeLabel.snp.bottom).offset(10)
+            $0.leading.trailing.equalToSuperview().inset(SizeLiteral.leadingTrailingPadding)
+        }
+    }
+
+    // MARK: - functions
```

Comment:
p4;

```suggestion
    // MARK: - func
```
사소하지만 요거도 가능할까요 ..??

Reply:
[0258ede](https://github.com/DeveloperAcademy-POSTECH/MacC-Team-Maddori.Apple/pull/27/commits/0258ede9052a762faff999a6bc6d04c00a59695e)
수정해서 반영해두었습니다 ~